### DISCUSSION TITLE
Antivirus won't let me download 0.4.6 update!

### DISCUSSION THREAD

**Discussion by tylere661** [OpenRCT2/OpenRCT2#20819] [Q&A]
I try to download the 0.4.6 update to openrct2 and my antivirus software keeps prohibiting me from downloading the files and also trying to extract the files denies access.  Any ideas on what to do?

**Comment by janisozaur** [comment]
Use either one or the other, but not both.

**Comment by 733737** [comment]
Use 7-zip
